Code/Framework/AzCore/AzCore/IO/Streamer/Scheduler.cpp

```diff
@@ -6,9 +6,11 @@
  *
  */
 
+#include <AzCore/IO/Streamer/Scheduler.h>
+
+#include <AzCore/IO/Streamer/FileRequest.h>
 #include <AzCore/Casting/numeric_cast.h>
 #include <AzCore/Debug/Profiler.h>
-#include <AzCore/IO/Streamer/Scheduler.h>
 #include <AzCore/std/containers/deque.h>
 #include <AzCore/std/sort.h>
```

Comment:
These should be kept in alphabetic order. If this was changed due headers needing to be called in a specific then that's preferably avoided.